### PR TITLE
Pass options to remarkable

### PR DESCRIPTION
**Pull request by juliankrispel**
Would be nice to pass down the options to Remarkable to have more control :)

### CODE REVIEW

**Comment by juliankrispel** [comment]
My use case for example is to parse html, and I prefer this converter over the others (was about to write my own when I found this)

**Comment by Rosey** [comment]
Thanks very much for the kind words and for contributing :) Also I can't believe I wasn't already passing options in, it seems like a no-brainer thing that should be added. I had the same frustration with existing draftjs markdown converters, needed more customization than they would easily allow.
 
One change though maybe? Instead of passing in `options` wholesale like this, since potentially `markdownToDraft` could also be accepting its _own_ options and I don't want to get painted into a corner where `remarkable` options and `markdownToDraft` options accept the same key with a different meaning and get people all confused, I'm wondering if we can change this to 

```javascript
const md = new Remarkable(options.remarkableOptions);
```
This would also kind of follow the naming convention that's already one line below that, where we reference `options.remarkablePlugins` :) 

Also, if you have the time, updating the README to include this little factoid, but if not I can do that in a separate PR later.

**Comment by juliankrispel** [comment]
Will do! Thx for the review 🙌

**Comment by juliankrispel** [comment]
@Rosey done, lmk if all good.

File: README.md [hunk 1]
```diff
@@ -100,3 +100,13 @@ var rawDraftJSObject = markdownToDraft(markdownString, {
   }
 });
 ```
+
+Since this module uses remarkable under the hood, you can also pass down options for the remarkable parser, simply add the property `remarkableOptions` to your options object. For example, let's say you wanted to parse html as well:
+
+```
```
Comment (Rosey) [inline README.md]: minor nit could specify `javascript` here so syntax highlighting happens on gh!

**Comment by Rosey** [comment]
one small comment but looks good otherwise :) thank you! Do you think you could squash any pr-comment related fixup commits together as one and force push before I merge? (Just because I don't think we need 3 separate commits for this change 🙃)

**Comment by juliankrispel** [comment]
@Rosey I had the same idea :D

**Comment by Rosey** [comment]
Yay awesome thank you. All merged. Will draft a release shortly!

**Comment by Rosey** [comment]
or do a release 🤔  I guess draft implies it won't actually be released. I will do a release shortly.

**Comment by juliankrispel** [comment]
@Rosey sweeeet! Thanks for the swift response! Btw, in case you haven't and fancy some draft-js company, join us on slack <3 - https://draftjs.herokuapp.com/

**Comment by Rosey** [comment]
I'm in there! I'm not very active though, more of a lurker mostly 😅

**Pull request merged**